src/main/java/Azir/Azir.java

```diff
@@ -0,0 +1,149 @@
+package Azir;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.Arrays;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import java.time.format.DateTimeParseException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Azir {
```

Comment:
I think you could add JavaDoc comments for your Azir class when you have the time?